README.md

```diff
@@ -24,8 +24,8 @@ If you have found an issue with the application, please [file a bug](https://bug
 
 ## Get in touch
 
-We're friendly! Talk to us on [IRC](https://webchat.freenode.net/?channels=snappy)
```

Comment:
Oh, IRC is still used for `snapd` development? Well you're the one who would know! :P